packages/react-positioning/src/usePopper.ts

```diff
@@ -427,7 +428,7 @@ export function usePopper(
     [resolvePopperOptions],
   );
 
-  if (process.env.NODE_ENV !== 'production') {
+  if (!isProd) {
```

Comment:
I quickly check and it seems that `packages/react-positioning/src/utils/getReactFiberFromNode.ts` leaks into bundle after this change.

| Fixture            | Minified  | GZIP     |
| ------------------ | --------- | -------- |
| usePopper (before) | 22.808 kB | 7.935 kB |
| usePopper (after)  | 23.72 kB  | 8.243 kB |

---

To repro:

```sh
yarn buildto @fluentui/react-positioning
yarn workspace @fluentui/react-positioning bundle-size
```

⬇⬇⬇
`packages/react-positioning/dist/bundle-size/usePopper.output.js`

![image](https://user-images.githubusercontent.com/14183168/141779678-8c21a68d-7d88-455c-bf57-95ed0791f10e.png)